models/pull.go

```diff
@@ -522,6 +522,9 @@ func GetLatestPullRequestByHeadInfo(repoID int64, branch string) (*PullRequest,
 
 // GetPullRequestByIndex returns a pull request by the given index
 func GetPullRequestByIndex(repoID, index int64) (*PullRequest, error) {
+	if index < 1 {
```

Comment:
For more experienced maintainers: Is `0` a valid PR ID or not?
If yes, this `if` should be updated.

Reply:
index starts with 1